.ci/sidetrack/Jenkinsfile

```diff
@@ -0,0 +1,239 @@
+// https://github.com/camunda/jenkins-global-shared-library
+@Library('camunda-ci') _
+
+String getAgent(String dockerImage = 'gcr.io/ci-30-162810/centos:v0.4.6', Integer cpuLimit = 4){
+  String mavenForkCount = cpuLimit;
+  String mavenMemoryLimit = cpuLimit * 2;
+  """
+metadata:
+  labels:
+    agent: ci-cambpm-camunda-cloud-build
+spec:
+  nodeSelector:
+    cloud.google.com/gke-nodepool: agents-n1-standard-32-netssd-preempt
+  tolerations:
+  - key: "agents-n1-standard-32-netssd-preempt"
+    operator: "Exists"
+    effect: "NoSchedule"
+  containers:
+  - name: "jnlp"
+    image: "${dockerImage}"
+    args: ['\$(JENKINS_SECRET)', '\$(JENKINS_NAME)']
+    tty: true
+    env:
+    - name: LIMITS_CPU
+      value: ${mavenForkCount}
+    - name: TZ
+      value: Europe/Berlin
+    resources:
+      limits:
+        cpu: ${cpuLimit}
+        memory: ${mavenMemoryLimit}Gi
+      requests:
+        cpu: ${cpuLimit}
+        memory: ${mavenMemoryLimit}Gi
+    workingDir: "/home/work"
+    volumeMounts:
+      - mountPath: /home/work
+        name: workspace-volume
+  """
+}
+
+pipeline {
+  agent none
+  parameters {
+    buildSelector(
+        name: 'copyArtifactSelector',
+        description: 'A build to take the artifacts from'
+    )
+    booleanParam defaultValue: true, description: '', name: 'STANDALONE'
+  }
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '5')) //, artifactNumToKeepStr: '30'
+  }
+  stages {
+    stage('Prepare') {
+      agent {
+        kubernetes {
+          yaml getAgent('gcr.io/ci-30-162810/centos:v0.4.6', 16)
+        }
+      }
+      when {
+        expression { params.STANDALONE == false }
+      }
+      steps {
+        copyArtifacts(projectName: "cambpm-jenkins-pipelines/${env.BRANCH_NAME}", fingerprintArtifacts: true, selector: buildParameter('copyArtifactSelector'), optional: true)
+        stash name: "platform-stash-runtime", includes: ".m2/org/camunda/**/*-SNAPSHOT/**", excludes: "**/qa/**,**/*qa*/**,**/*.zip,**/*.tar.gz"
+        stash name: "platform-stash-qa", includes: ".m2/org/camunda/bpm/**/qa/**/*-SNAPSHOT/**,.m2/org/camunda/bpm/**/*qa*/**/*-SNAPSHOT/**", excludes: "**/*.zip,**/*.tar.gz"
+      }
+    }
+    stage('UNIT DB tests') {
+      matrix {
+        axes {
+          axis {
+            name 'DB'
+            values 'h2'// TODO 'cockroach'
+          }
+          axis {
+            name 'PROFILE'
+            values 'engine-unit', 'engine-unit-authorizations', 'webapps-unit', 'webapps-unit-authorizations', 'check-sql', 'upgrade-db', 'instance-migration', 'old-engine', 'rolling-update', 'large-data-tests'
+          }
+        }
+        when {
+          expression {
+            return withLabels('all-db','cockroachdb') // TODO
+          }
+          beforeAgent true
+        }
```

Comment:
When coupling this with the condition for triggering in the main `Jenkinsfile`, it seems that this pipeline may be triggered, but nothing will be executed. E.g. if the `default-build`, `rolling-update` or `migration` labels are used, nothing will be triggered for this pipeline. Is this the desired effect since the idea is to make it a pipeline for slow jobs?